src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -441,6 +441,11 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
       event.stopPropagation();
     }
 
+    if (event.key && event.key === 'Enter') {
+      // Regain focus after multiselect, so we can further type
+      setTimeout(() => this._focus(), 200);
```

Comment:
would it also work without setting a specific delay, i.e. the following?
```suggestion
      setTimeout(() => this._focus());
```

Reply:
Yes that seems to work, at least in chromium, I'll try it in firefox.

Reply:
Tested it in firefox, work's there, too.